### DISCUSSION TITLE
Fetch Hyperlinks

### DISCUSSION THREAD

**Discussion by matheusmaus** [TheAxeDude/ZiniGo#14] [Ideas]
Hi @TheAxeDude,

Just want to make a suggestion, some digital magazines now are providing additional content via hyperlinks. To have the full content of the magazine, you're required to have it, most of them are related to videos or a material that needs to be printed.

Perhaps is possible to when it's fetching the page, you can generate an index of links that's in that issue, like the page that is in it, the clickable name, and the link itself. Or even commenting on the PDF to aggregate the links. 

Thanks

**Comment by TheAxeDude** [comment]
Hi there @matheusmaus 

This is a very interesting suggestion. The JSON that comes back for each page does include links in it, so I will take a look at that, and if they look reasonable, we can look at adding them. I like the idea of having a "comment" on each page with the links.

Cheers
